mailbox/api/pom.xml

```diff
@@ -53,6 +53,14 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-configuration</groupId>
+            <artifactId>commons-configuration</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
```

Comment:
I remember we deny to use this dependency, but I could not remember which dependency to replace